hash/vtech_innotab_cart.xml

```diff
@@ -16,17 +16,21 @@ license:CC0-1.0
 	Language:
 	********0 = US / Canada / Hong Kong / Japan
 	********1 = Unused
+	*******18 = China
 	*******13 = Denmark
 	********2 = Canadian English
 	*******22 = Spain / Mexico / Latin America
 	********3 = UK
 	*******23 = Netherlands
 	********4 = Germany
+	*******41 = Japan
```

Comment:
Where are you actually getting this from?  If it’s really “unconfirmed” as the PR title states, it has no place here.